i18n/es/docusaurus-plugin-content-docs/current/Images/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
-  "label": "🖼️ Image Prompting",
+  "label": "🖼️ Parámetros para imágenes",
```

Comment:
I think we should keep the word prompt here

Reply:
I'll leave 'image prompting' then. Appreciate the suggestion.